client/src/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, withRouter } from 'react-router-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+import { Auth0Provider } from './react-auth0-wrapper';
+import config from './auth_config';
+
+// Routes the user to the right place
+// after login
+const onRedirectCallback = appState => {
+  window.history.replaceState(
+    {},
+    document.title,
+    appState && appState.targetUrl
+      ? appState.targetUrl
+      : window.location.pathname
```

Comment:
This line could be changed depending on where the user will be redirected to. This depends on user flows.